scripts/find_nearby_airplanes.py

```diff
@@ -0,0 +1,117 @@
+import itertools
+from argparse import ArgumentParser
+from datetime import datetime
+
+import geohash
+from geopy.distance import geodesic
+from influxdb import InfluxDBClient
+
+
+def main():
+    parser = ArgumentParser(
+        description="Writes some example FIXM data to InfluxDB")
+    parser.add_argument("--hostname", type=str, default="localhost",
+                        help="The hostname of InfluxDB")
+    parser.add_argument("--port", type=int, default=8086,
+                        help="The port to connect to InfluxDB on")
+    parser.add_argument("--username", type=str, default="root",
+                        help="The username to authenticate with InfluxDB")
+    parser.add_argument("--password", type=str, default="root",
+                        help="The password to authenticate with InfluxDB")
+    parser.add_argument("--database", type=str, default="fixm",
+                        help="The name of the database to write to")
+    parser.add_argument("--minimum-distance", type=float, default=10.0,
+                        help="The minimum allowed distance between aircraft, "
+                             "in miles")
+    parser.add_argument("--maximum-time-difference", type=float, default=5.0,
+                        help="The maximum allowed time difference between "
+                             "comparable records")
+    parser.add_argument("--duration", type=str, default="1m",
+                        help="How far back to look for nearby aircraft. "
+                             "Format the duration as a number followed by 'h' "
+                             "for hours, 'm' for minutes, or 's' for seconds.")
+    args = parser.parse_args()
+
+    # Establish a connection to InfluxDB
+    client = InfluxDBClient(args.hostname, args.port,
+                            args.username, args.password,
+                            args.database)
+
+    # Get all geohashes with aircraft currently in them
+    result = client.query(f"SELECT geohash FROM fixm.autogen.TH "
+                          f"WHERE time > now() - {args.duration}")
+    result = result.get_points("TH")
+    ghashes = set(r["geohash"] for r in result)
```

Comment:
Use set comprehension
```suggestion
    ghashes = {r["geohash"] for r in result}
```